tests/regression/lp-1978005/do-not-track/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: do-not-track 
+version: 1
+architectures: [all]
+apps:
+  track:
+    command: bin/sh
+  do-not-track:
+    command: bin/sh
+    do-no-track: true
```

Comment:
Typo here, `do-no-track` rather than `do-not-track`.

Reply:
Thanks, fixed.